core/os/condition_variable.h

```diff
@@ -63,5 +67,14 @@ class ConditionVariable {
 		condition.notify_all();
 	}
 };
+#else
```

Comment:
Like I suggested elsewhere, in this situation I'd add empty lines around the `#ifdef THREADS_ENABLED` / `#else` / `#endif // THREADS_ENABLED` to make it easier to see where the two branches start and end.

Same for other similar conditional class definitions.